docker/swarm/dev/swarm-aws.yml

```diff
@@ -0,0 +1,576 @@
+version: '3.6'
+
+networks:
+
+  proxy:
+    external: true
+
+  backend:
+    driver: overlay
+    external: false
+
+volumes:
+
+  traefik-volume:
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: '/mnt/efs/traefik-volume'
+
+  zookeeper-1-volume:
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: '/mnt/efs/zookeeper-1-volume'
+  zookeeper-2-volume:
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: '/mnt/efs/zookeeper-2-volume'
+  zookeeper-3-volume:
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: '/mnt/efs/zookeeper-3-volume'
+
+  kafka-1-volume:
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: '/mnt/efs/kafka-1-volume'
+  kafka-2-volume:
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: '/mnt/efs/kafka-2-volume'
+  kafka-3-volume:
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: '/mnt/efs/kafka-3-volume'
+
+  kafka-streams-volume:
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: '/mnt/efs/kafka-streams-volume'
+
+  kafka-connect-volume:
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: '/mnt/efs/kafka-connect-volume'
+
+  control-center-volume:
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: '/mnt/efs/control-center-volume'
+  control-center-config-volume:
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: '/mnt/efs/control-center-config-volume'
+
+  mongodb-volume:
+    driver: local
+    driver_opts:
+      type: 'none'
+      o: 'bind'
+      device: '/mnt/efs/mongodb-volume'
+
+services:
+
+  # ///////////////////////////////////////////////////////
+  # Front Services
+  # ///////////////////////////////////////////////////////
+
+  traefik:
+    image: enkryptio/traefik:0.1.4
+    networks:
+      - proxy
+      - backend
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - traefik-volume:/var/lib/traefik
+    ports:
+      - 80
+      - 8080
+    environment:
+      DEBUG: 'true'
+      LOG_LEVEL: INFO
+      SWARM_MODE: 'true'
+    deploy:
+      placement:
+        constraints:
+          - node.role == manager
+      replicas: 1
```

Comment:
Set replicas to 0 in all of the docker services by default except zookeeper-1/2/3